src/components/MultipleFilter/internal/FilterCard/FilterCard.tsx

```diff
@@ -44,15 +44,26 @@ export const FilterCard: React.FunctionComponent<FilterCardProps> = (
     width,
     currentReferredFilters,
     sectionTitle = "Section",
-    conditionTitle = "Condition",
+    conditionTitle = "",
   } = props;
 
   const theme = useTheme();
   const [section, setSection] = React.useState<string | undefined>();
   const [condition, setCondition] = React.useState<string | undefined>();
-  const [selectedFilter, setSelectedFilter] = React.useState<FilterType>();
+  const [selectedFilter, setSelectedFilter] = React.useState<
+    FilterType | undefined
+  >(
+    selectedFilterPack?.filters.length === 1
+      ? selectedFilterPack.filters[0]
+      : undefined,
```

Comment:
これもつまり isMultipleFilters なのではと思いましたがどうでしょう？
（ isMultipleFilters を先に宣言する必要ありますが）
```suggestion
    isMultipleFilters
      ? undefined
      : selectedFilterPack.filters[0]
```

Reply:
ちなみに万が一 filters が0件の場合は結果的に undefined が初期値になる